### PR TITLE
MC: Bump rmc version in aggregator

### PR DESCRIPTION
**Pull request by kostekIV**

### CODE REVIEW

File: finality-aleph/Cargo.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ license = "Apache 2.0"
 aleph-bft = "0.18.1"
 aleph-bft-rmc = "0.4.0"
 aleph-primitives = { package = "primitives", path = "../primitives" }
-aleph-aggregator = { package = "aggregator", path = "../aggregator" }
+aleph-aggregator = { package = "aggregator", git = "https://github.com/Cardinal-Cryptography/aleph-node.git", tag = "aggregator-rmcv0.4" }
```
Comment (timorl) [inline finality-aleph/Cargo.toml:12]: Uh, the tag should probably be versioned by the version of aggregator itself, rather than its dependencies.

**Review by timorl — APPROVED** [review]
Looks good, does it work? Also, is aggregator being compiled and tested as part of the pipeline currently?

File: finality-aleph/Cargo.toml (hunk 1 above)
Comment (kostekIV) [reply finality-aleph/Cargo.toml:12]: will rename

**Comment by kostekIV** [comment]
> Looks good, does it work? Also, is aggregator being compiled and tested as part of the pipeline currently?

let me double check it, it should be as as it is part of the workspace

File: finality-aleph/Cargo.toml (hunk 1 above)
Comment (kostekIV) [reply finality-aleph/Cargo.toml]: done

**Comment by kostekIV** [comment]
> > Looks good, does it work? Also, is aggregator being compiled and tested as part of the pipeline currently?
> 
> let me double check it, it should be as as it is part of the workspace

yep it is compiled and tested in pipelines

**Review by Marcin-Radecki — APPROVED** [review]

**Pull request merged**